es-core/src/components/VideoComponent.cpp

```diff
@@ -221,6 +212,15 @@ void VideoComponent::applyTheme(const std::shared_ptr<ThemeData>& theme, const s
 	if (elem->has("showSnapshotDelay"))
 		mConfig.showSnapshotDelay = elem->get<bool>("showSnapshotDelay");
 
+	setRotation(0.0);
```

Comment:
I dont think this is a good idea. Later an animation can have size change and rotation and if the order is rotation and size change then rotation won't work.